src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="block">
+      <div class="stopwatch stopwatch--speed-up">
+        <div class="stopwatch__min-hand"></div>
+        <div class="stopwatch__sec-hand"></div>
+        <div class="stopwatch__circle"></div>
+      </div>
+    </div>
   </body>
```

Comment:
The `body` tag should be closed after all the content has been listed. It seems like there is a missing closing tag for `body`.